torch/csrc/jit/script/compiler.cpp

```diff
@@ -7,6 +6,7 @@
 #include <torch/csrc/jit/passes/canonicalize.h>
 #include <torch/csrc/jit/passes/constant_pooling.h>
 #include <torch/csrc/jit/passes/lower_tuples.h>
+#include <torch/csrc/jit/script/compiler.h>
```

Comment:
I think our convention is to have the corresponding header for a cpp compilation unit appear first in the includes